main/mainworker.cpp

```diff
@@ -12455,7 +12448,7 @@ bool MainWorker::SwitchScene(const uint64_t idx, std::string switchcmd)
 			_log.Log(LOG_NORM, "Activating Scene/Group Device: %s (%s)", DeviceName.c_str(), lstatus.c_str());
 
 
-			int ilevel = maxDimLevel - 1;
+			int ilevel = maxDimLevel - 1; // Why -1?
```

Comment:
This had something to do with KAKU dimmers... do you own those ?
Else i have to dig into my boxes to find one...
I believe it will blink the light or something... but best to test

Reply:
No, sorry, I don't have any KAKU dimmers.
I have some Proove dimmers which blink if ON is sent twice (to allow dimming with an ON/OFF button), not sure if this is what you mean?

Reply:
i think those are the same...

Reply:
In case the switch is a dimmer, ilevel is overwritten by the block a few lines down:
```
if (lstatus == "On")
{
	lstatus = "Set Level";
	float fLevel = (maxDimLevel / 100.0f)*level;
	if (fLevel > 100)
		fLevel = 100;
	ilevel = round(fLevel);
} 
```
Anyhow, I did some test with my Proove dimmer, blinking happens if sending "ON" when already turned on.
Nothing special happens when set to max dimlevel (15).